typescript/src/3_functions/index.ts

```diff
@@ -69,8 +69,9 @@ export default (() => {
 
     return lettersAndPoints.reduce((computedScore, pointsTuple) => {
       const [letters, score] = pointsTuple;
-      if (letters.split('').find((ll) => ll === letter)) {
-        return computedScore += score;
+      console.log(pointsTuple)
+      if (String(letters).split('').find((ll) => ll === letter)) {
```

Comment:
No need of casting letters to String here. Instead, set the type for lettersAndPoints variable.